Second_Batch/vishwa/commands.md

```diff
@@ -0,0 +1,49 @@
+1. cat : create and edit file
+
+2. touch : create a empty files and modifies time
+
+3. cal : calendar
+
+4. cp : copies file
+
+5.mkdir : make the directories
+
+6. date : date
+
+7. mv : move files or directories
+
+8. rm : removes the file
+
+9. rmdir : remove directories
+
+10. info : shows the information abouth the commands
+
+11. pwd : present working directories
+
+12. who : list of users
+
+13. ls : list of contents
+
+14. du : disk usage
+
+15. whoami : shows the currently logged in user
+
+16. clear: clear terminal
+
+17. df : disk free
+
+18. banner : prints in banner format
+
+19. figlet : prints the line
+
+20. find : find files or directories
+
+21. man : manual for commands
+
+22. which : shows executable path of environment variable
+
+23. su : switch user/super user
+
+24. sudo : super user do/ run as administrator
+
+25. cd : chande the directories
```

Comment:
correct the spelling on line no. 25 from chande to change

Reply:
Done